config/config.exs

```diff
@@ -57,11 +57,9 @@ config :ueberauth, Ueberauth.Strategy.Cognito,
   user_pool_id: {System, :get_env, ["COGNITO_USER_POOL_ID"]},
   aws_region: {System, :get_env, ["COGNITO_AWS_REGION"]}
 
-env_name =
-  case System.get_env("ENVIRONMENT_NAME") do
-    nil -> Mix.env()
-    env -> String.to_existing_atom(env)
-  end
+config :sentry,
+  enable_source_code_context: true,
+  root_source_code_path: File.cwd!()
```

Comment:
I'm a little wary about having this piece of config evaluate at compile time, in case the file structure or location that the application runs in changes between when it's compiled in GH Actions and when it runs in AWS.

At the same time, it might be better to evaluate here, during build, since we're looking for the root source code path? Hmmm.

If you're sure it will work as intended, or would be easy to catch and fix if it doesn't, looks good to me!

Reply:
Oh uhhh hm! That's good insight. I realize this isn't needed anywhere but for the released app, so back to the releases.exs it goes.